packages/chain-adapters/src/cosmossdk/CosmosSdkBaseAdapter.ts

```diff
@@ -57,6 +58,16 @@ export const assertIsValidatorAddress = (validator: string, chainId: CosmosSdkCh
   }
 }
 
+type ConfirmationSpeed = 'slow' | 'average' | 'fast'
```

Comment:
Could reuse `FeeDataKey` here?

Reply:
yeah good call, make it pretty here: https://github.com/shapeshift/lib/pull/1227/commits/eb271d78e0e6012261611c6e02fb0da8cb1efd7e